<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="modulation_frequency_group">
     <property name="title">
      <string>Modulation frequency</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QDoubleSpinBox" name="modulation_frequency">
          <property name="decimals">
           <number>5</number>
          </property>
          <property name="stepType">
           <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>MHz</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Set modulation frequency to 0 to display the DC signal.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="modulation_amplitude_group">
     <property name="title">
      <string>Modulation amplitude</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QDoubleSpinBox" name="modulation_amplitude">
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="maximum">
           <double>2.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="stepType">
           <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Vpp</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Sweep speed</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QComboBox" name="sweep_speed">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>3.8 kHz</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1.9 kHz</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>954 Hz</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>477 Hz</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>238 Hz</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>119 Hz</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>59 Hz</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>30 Hz</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>15 Hz</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>7.5 Hz</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3.7 Hz</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1.9 Hz</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>0.93 Hz</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>0.47 Hz</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>0.23 Hz</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>0.12 Hz</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Sweep amplitude and offset</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Use the controls &lt;span style=&quot; font-style:italic;&quot;&gt;Sweep amplitude&lt;/span&gt; and &lt;span style=&quot; font-style:italic;&quot;&gt;Position &lt;/span&gt;above the plot widget&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="spectroscopyTabs">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="spectroscopy_channel_1_page">
      <attribute name="title">
       <string>Channel 1</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_20">
       <item>
        <widget class="SpectroscopyChannelAPanel" name="widget_4" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="spectroscopy_channel_2_page">
      <attribute name="title">
       <string>Channel 2</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_19">
       <item>
        <widget class="QLabel" name="spectroscopyBPanelDisabled">
         <property name="text">
          <string>Enable dual channel spectroscopy in &quot;General&quot; tab in order to use the second channel.</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignJustify|Qt::AlignTop</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="SpectroscopyChannelBPanel" name="spectroscopyBPanel" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SpectroscopyChannelBPanel</class>
   <extends>QWidget</extends>
   <header location="global">spectroscopy_panel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SpectroscopyChannelAPanel</class>
   <extends>QWidget</extends>
   <header location="global">spectroscopy_panel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
